### ISSUE TITLE
More Informative Result Table

### ISSUE THREAD

**Issue by DarioS** [jyyu/ScreenBEAM#3]
The result table has gene-level statistical results. However, once the important genes are identified, the biologist would like to know which of the guide RNAs for each gene contributed the most to the test statistic, so that it could reduce the amount of experimental work to do for laboratory validation using 1 guide RNA at a time. Choosing the strongest signal increases the chances of experimental validation and reduces the need to test more guide RNAs if the first one tested works well. Could the result table be enhanced with extra details to allow prioritisation of guide RNAs for experimental tasks?

**Comment by jyyu** [comment]
Good point Dario. Actually there is a hidden function "DRAshRNALevel" in the package, which you can use to get individual sh/sgRNA level statistics and then select the top hairpins or guides for the genes prioritized from the gene-level analysis. I will try to write another wrap function to make this one visible and document it on GitHub. Thank you.

**Comment by yangyulin0809** [comment]
Hi jyyu,

Have you finished the wrap function to display the guide RNA/shRNA details in the result?
I'm interested in ScreeBEAM, and want to use it to analyze my own sequence data. And I also think the information in guide RNA/shRNA leve is very important for me.
Look forward to your reply!

Many thanks!
Yulin

**Comment by jyyu** [comment]
Thanks for your interest Yulin. We plan to release it in our new version, which has many more features. In the meanwhile, you can try the function I mentioned above and let me know if running into any problems.

**Comment by yangyulin0809** [comment]
Just add "DRAshRNALevel=TRUE" in the code? I'm a little confused, because I met an error as following:
![image](https://cloud.githubusercontent.com/assets/16235664/25415294/65ef272a-2a69-11e7-837f-94f4a491e3a1.png)

**Comment by DarioS** [comment]
It's a function - not a parameter. You need to manually open utilities.R in the R folder of the package and then search the text file for `DRAshRNALevel`. Once you find it, you'll see the function declaration, which shows what parameters it accepts. Some of the options lack clear explanations, such as

```
do.restand: whether do restand in the analysis
```

so it's best to leave them at their default values if you're unsure.

**Comment by yangyulin0809** [comment]
My R version is R-3.3.3. I installed ScreenBEAM two days ago. Do you mean the function DRAshRNALevel belongs to ScreenBEAM? Once I load the package ScreenBEAM, I can use the function DRAshRNALevel ?  Actually I can't find the text file for DRSshRNALevel, because I'm not very familiar with R. It's very kind of you if you can add the function DRAshRNALevel in the examples. 
Many Thanks!

**Comment by DarioS** [comment]
It's not exported from the package so it isn't available directly to you if you typed the command `library(ScreenBEAM)`. The only functions currently intended for end-user usage are listed in the file named `NAMESPACE`. Regardless, you can still use the function by typing `ScreenBEAM:::DRAshRNALevel(aSet, conditionsNames, controlsNames)` if you created variables named `aSet`, `conditionsNames`, and `controlsNames`, but will need to open the source code to read the documentation of what other parameters it has (i.e. there is no Rd file for it).
